payment/key_test.go

```diff
@@ -54,8 +68,10 @@ func TestSecretKey(t *testing.T) {
 	if err != nil {
 		t.Error("failed to generate a secret key: ", err)
 	}
+	encryptedBytes, nonceBytes, err := cryptography.EncryptMessage([]byte(k.SecretKey))
 
-	k.EncryptedSecretKey, k.Nonce, err = encryptSecretKey(k.SecretKey)
+	k.EncryptedSecretKey = fmt.Sprintf("%x", encryptedBytes)
+	k.Nonce = fmt.Sprintf("%x", nonceBytes)
```

Comment:
i would add another test that tests the output of this against a known value, using a fixed secret key instead of a random string. even better if it uses one of the official secretbox test vectors: https://pynacl.readthedocs.io/en/stable/vectors/.

Reply:
i guess to test encryption though you need to be able to fix the nonce value instead of randomly generating it in the test. you can probably still use test vectors for decryption tests though (since the nonce is fed as input).

Reply:
Here's some tests for just the `encryption.go` file using a static string 😄 https://github.com/brave-intl/bat-go/pull/347/commits/6396ea948031c9eb0580a66442e3e3342bb731de